app/views/unattended/provisioning_templates/PXELinux/preseed_default_pxelinux.erb

```diff
@@ -28,6 +28,10 @@ oses:
   else
     options << 'console-setup/ask_detect=false console-setup/layout=USA console-setup/variant=USA keyboard-configuration/layoutcode=us localechooser/translation/warn-light=true localechooser/translation/warn-severe=true'
   end
+  if @host.provision_interface.vlanid
+    vlan_opt = "netcfg/use_vlan=true netcfg/vlan_id=#{@host.provision_interface.vlanid}"
+    options << "#{vlan_opt}"
+  end
```

Comment:
Can you make this just one line? Also squash everything into a single commit if possible. I haven't tested but it looks good.

Reply:
I assume you still want the `if` / `end` syntax so it's consistent with the other conditiojnals in this block.